tests/rptest/tests/rpk_acl_test.py

```diff
@@ -327,22 +327,23 @@ def test_schema_registry_acl(self):
         assert len(acl_list_all['matches']) == 0
 
         # ACL with SR + Kafka = 4 in total.
-        sr_kafka_acl = RPKACLInput(allow_principal=["panda"],
-                                   topic=["foo"],
-                                   registry_subject=["foo-value"],
-                                   operation=["read", "write"],
-                                   resource_pattern_type="literal")
+        sr_kafka_acl = RPKACLInput(
```

Comment:
I am curious why we replaced the tests instead of adding another test case ? @r-vasquez

Reply:
Oh I see, no. not a blocker